specification/nginx/Nginx.Management/ApiKeys.tsp

```diff
@@ -0,0 +1,109 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using Rest;
+using Azure.ResourceManager;
+
+namespace Nginx.NginxPlus;
+
+/**
+ * API key resource returned by the service.
+ */
+@parentResource(NginxDeployment)
+model ApiKey is ProxyResource<ApiKeyResponseProperties> {
+  ...ResourceNameParameter<ApiKey>;
+}
+
+/**
+ * Request payload for creating or updating an API key.
+ */
+@parentResource(NginxDeployment)
+@resource("apiKeys")
+model ApiKeyRequest is ProxyResource<ApiKeyRequestProperties> {
+  ...ResourceNameParameter<ApiKey>;
+}
+
+/**
+ * Properties returned for an API key resource.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "We intentionally do not include provisioning state for API keys."
+model ApiKeyResponseProperties {
+  /**
+   * Hint for the secret (read-only)
+   */
+  hint: string;
+
```

Comment:
What is a hint for the secret?

Reply:
Who creates the secret hint? How? I dont' see it in the configurations create example.